tests/sqlparser_snowflake.rs

```diff
@@ -24,6 +41,120 @@ fn test_snowflake_create_table() {
     }
 }
 
+fn get_from_section_from_select_query(query: &str) -> Vec<TableWithJoins> {
```

Comment:
Can't you use `verified_only_select(sql).from` instead of creating a new helper?

Reply:
It's a problem to use it because we remove the information about [ we ignore the information about  the redundant parenthesis in the AST ](https://github.com/ballista-compute/sqlparser-rs/issues/223#issuecomment-666334528)  , so the input sql will   be different than the output .